source/_posts/Citycross.md

```diff
@@ -0,0 +1,29 @@
+Hustá síť ulic implikuje velké množství křižovatek a tedy ještě víc možností, kudy se vydat. Nikdy není třeba opakovat tu samou trasu dvakrát. Pohledem do osobní heatmapy na Stravě nebo do aplikace Wandrer[1] navíc jde snadno najít ulice a místa, kudy jsem ještě nikdy neběžel. Každý běh tak může být expedicí. Expedicí za místy, která jsou perfektně normální, ale zároveň každé unikátní.
+
+> Tu čtyřprocentní plastika mezi paneláky. Támhle funkcionalistická vila o které jsem nikdy neslyšel a v zaplivaném parčíku za ní kvetou bledule. Když se sehnu a protáhnu pod šípky, nasaju vůni pražců a po štěrku podél kolejí přiběhnu na nástupiště. Na vlak nečekám. Radši se po schodech vrhnu do proudu cestujících v podchodu. V nádražní hale voní kafe z automatu. Sevřen na eskalátorech odpočítávám patnáctivteřinovou pauzu abych se s o to větší výbušností odrazil vstříc kočičím hlavám, které prověří mé kotníky. Skokem přes červenobílé zábradlí. Oční kontakt s řidičem předejde kontaktu s kapotou. Dík, že dáváš bacha! Voda z pítka hasí žízeň a smáčí kšiltovku. Asi pět kilometrů za mnou. Kolik jich ještě namotám do uliční sítě je mi zatím tajemstvím.
+
+Občas se mi zachce exotiky. To pak do kapsy přihodím jízdenku a nechám se odvézt metrem nebo tramvají někam daleko. Třeba na tu periferii, kam se jinak jezdí jen z donucení. Nechám se vyplivnout na konečné a s pomocí podvědomé buzoly trasuju cestu domů. Napříč hlavními tahy. Když uzřu lákavou odbočku, tak do ní zahnu. Zásadně se nevracím stejnou cestou zpět ač to občas stojí botu zabořenou v bahně, trní zapíchané ve stehnech nebo bundu prodřenou o plot, jak se přes něj sápu. A když už spojujeme běh a MHD, zkoušeli jste někdy vyběhnout eskalátory na Náměstí Míru v protisměru?
+
+Ploty a zdi bývají problém čím dál tím častěji. Někdo možná ještě pamatuje průchody a vnitrobloky běžně otevřené. Já možná budu vyprávět dětem, jak se kdysi stavěla bydlení, která se neshovávala za plot s bránou a recepcí. Prostupnost je půl klíče k městu, kde běh dělá radost. Málo co potěší tak, jako na zdánlivě slepém konci průmyslového areálu najít díru v plotě a za ní vyšlapanou pěšinu. Jestli se někdy radikalizuju, tak v tílku a s nůžkama v ruce. Budu prostřihávat (staro)nové průchody skrz ploty, které stojí akorát všem nazmar. Snad mi to kdyžtak rozmluví kamarádi.
+
+A těch je ve městě vždy dostatek. Byla by hloupost ignorovat, že ve městě se městná spousta lidí a nevzít nějaké z nich s sebou na výběh. Citycross jako drzý a spontánní řez uličním prostorem není udržitelný s rostoucí velikostí skupiny. Někde mezi dvěma a třema běžci leží optimum, ve kterém se jejich individuální kreativita a spontánnost doplňují. Můj oblíbený režim spočívá v druhém běžci zezadu dávajícím povely tomu prvnímu, jakým směrem se má vydat. Druhý se může soustředit na vymýšlení trasy, zatímco slepě kopíruje kroky prvního, který má volné nohy na prošlapávání cesty.
```

Comment:
nit: `že ve městě se městná spousta lidí` tam to "městě městná" je hodně `měs` :)) možná to byl záměr, tak nebudu podsouvat nějaké návrhy a dávám jen jako suggest